docs.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: Flutter Documentation
+permalink: /docs/
+---
+
+<ul class="cards">
+{% for card in site.data.docs_cards %}
+	<li class="cards__item">
+	    <div class="card">
+		    <h3 class="catalog-category-title"><a class="action-link" href="{{card.url}}">{{card.name}}</a></h3>
+		    <p>{{card.description}}</p>
+		    <div class="card-action">
+		        <a class="action-link" href="{{card.url}}">VISIT</a>
+		    </div>
+		</div>
+	</li>
+{% endfor %}
+</ul>
+
+&nbsp;
+
+
+## New to Flutter?
+
+* [Write Your First Flutter App](http://example.com)  
+  A walkthrough on creating a Flutter app that generates names for startup companies.
+
+* [Flutter for Android Developers](/flutter-for-android/)  
+  Tips for Android developers.
+
+* [HTML/CSS patterns](/web-analogs/)  
+  Tips for web developers showing how HTML and CSS syntax maps to Flutter widgets.
+
+* [Building Layouts in Flutter](/tutorials/layout/)  
+  Walkthrough showing how to create layouts in Flutter, where everything is a widget.
+
+* [Add Interactivity to Your Flutter App](/tutorials/interactive/)  
+  Walkthrough showing how to add a stateful widget to your app.
+
+* [A Tour of the Flutter Widget Framework](/widgets-intro/)  
+  A short, conceptual tour through the Flutter framework.
+
+&nbsp;
+
+
+## Want to skill up?
```

Comment:
will this be easy to understand for non-native speakers?

Reply:
I _think_ so. 

@Sfshaza, what are our options to rephrase?